soh/soh/Enhancements/randomizer/randomizer_check_tracker_window.h

```diff
@@ -0,0 +1,25 @@
+#include <libultraship/libultraship.h>
```

Comment:
Why isn't this just in `soh/soh/Enhancements/randomizer/randomizer_check_tracker.h`?

Reply:
all of it or just the include? i don't remember exactly why this has it's own file (i remember running into a ton of header include issues with c vs cpp when refactoring all the windows to be `LUS:GuiWindow`s) but i don't remember if this one was specifically an issue. I can try moving it back in there at some point and see if that jogs my memory

Reply:
this time it seemed to just work when i moved it over to the other header https://github.com/HarbourMasters/Shipwright/pull/2881/commits/81c53add1dc8eb8f2ad53662ff2e320385e99b42